Gemfile

```diff
@@ -0,0 +1,8 @@
+source 'https://rubygems.org'
+gem 'rspec', '~>3.7.0'
+gem 'date', '~>3.1.0'
+gem "erb", '~>2.2.0'
+gem "json", '~>2.5.1'
+gem "net-http", '~>0.1.1'
+gem "uri", '0.10.0'
+gem "bundler-audit", '~>0.9.0.1'
```

Comment:
🥳 this looks awesome!

Just a couple of minor things - might be worth having consistent quotes in here. We will probably look to introduce a linter (rubocop) later and it will have a tantrum about it anyway.

Also, could we specify the ruby version as well? Means things like rbenv can detect it and install https://bundler.io/gemfile_ruby.html

Reply:
Fixed